Automattic/woocommerce-payments
# Use client-side navigation for links within app

#### Changes proposed in this Pull Request

Convert ordinary `<a href>` links to `<Link href>` internal links, to avoid an unnecessary page reload when possible.

Deposit links (https://github.com/Automattic/woocommerce-payments/commit/2d17608811d4a65d2581ea4ef6135e911352f6d5):
<img width="715" src="https://user-images.githubusercontent.com/1867547/122830589-8d36ed00-d2b6-11eb-95ee-cdf82437666d.png">

Customer links, outside WCPay but inside WC Admin single-page app (https://github.com/Automattic/woocommerce-payments/commit/b08308c86bf083fe6f9e2383a8c9a27f7df8a340):
<img width="635" src="https://user-images.githubusercontent.com/1867547/122830590-8dcf8380-d2b6-11eb-8904-2dec43c121ad.png">

Deposit & dispute links (https://github.com/Automattic/woocommerce-payments/pull/2290/commits/67873e2084f081f3bfbba6ad3bc6175119dbeda9):
<img width="532" src="https://user-images.githubusercontent.com/1867547/122830588-8d36ed00-d2b6-11eb-8d0d-b62ad1a9ddb5.png">

#### Testing instructions





Test links noted above, and verify that they load a new screen within the existing page session instead of reloading the page.

-------------------
